semantics.json

```diff
@@ -342,6 +342,14 @@
         "default": true
       },
       {
+        "name": "randomizeOnRetry",
+        "type": "boolean",
```

Comment:
I think there are cases where the order of the alternatives is deliberate. I.e. consistently ordered by something (size/alphabetically/chronologically) throughout a question set, or the options are always the same for many questions(word classes for instance) and randomizing them for each question would only be frustrating so I don't think we should remove that feature.